vnfs/libexec/wwmkchroot/include-suse

```diff
@@ -1,6 +1,4 @@
-REPO_NAME="SLES12-12-0-sp1"
-ZYP_CONF="/root/yum-ww.conf"
-ZYP_CMD="zypper -c $CHROOTDIR/$ZYP_CONF --root $CHROOTDIR -n --gpg-auto-import-keys"
+ZYP_CMD="zypper --root $CHROOTDIR -n --gpg-auto-import-keys"
```

Comment:
I assume zypper chroots first then loads it config file so specifying the config path isn't needed here?

Reply:
That is a correct assumption. The zypper tool is frequently used on chroot base paths, and we here at SUSE like to minimize the number of options needed in our tools. ;-)